spec/models/ring_spec.rb

```diff
@@ -4,14 +4,16 @@
   describe "#link" do
     it "adds a new redirection while preserving the ring" do
       new_redirection = build(:redirection)
```

Comment:
It's because of callbacks for integrating into the ring, yes. It's not really because of hitting the DB.